core/GlobalState.cc

```diff
@@ -578,6 +581,14 @@ void GlobalState::initEmpty() {
     }
     method.data(*this)->resultType = Types::untyped(*this, method);
 
+    // Synthesize <Magic>#<getEncoding>() => Encoding
```

Comment:
```suggestion
    // Synthesize <Magic>.<getEncoding>() => Encoding
```

Reply:
(most of these comments are using the wrong symbol; don't feel like you have to update all of them unless you want to)